bin/kano-tracker-ctl

```diff
@@ -15,7 +15,7 @@ Usage:
   kano-tracker-ctl refresh [-w]
   kano-tracker-ctl session (start|end) <name> <pid>
   kano-tracker-ctl session log <name> <started> <length>
-  kano-tracker-ctl session run <name> <command>
+  kano-tracker-ctl session run <name> <command> [<cmdargs>]
```

Comment:
I think that instead we should do `kano-tracker-ctl session run <name> <command> [<cmdargs>...]` so that it can take as many arguments as we wish, and then `args['<cmdargs>']` is a list so you'll need to do

```python
commandline += ' '.join(args['<cmdargs>'])
```

(or maybe `' '.join(map(str, args['<cmdargs>']))` if it could be something which isn't a string)

Reply:
Good point Tom, thanks! Fixed it